container/ubuntu16_04/builders/rbe-ubuntu16_04/BUILD

```diff
@@ -53,6 +53,7 @@ toolchain_container(
         "-c",
         "/bin/bash",
     ],
+    installables_tar="@ubuntu16_04_debian_pkgs//:ubuntu16-04-debian_pkgs.tar",
```

Comment:
This would break the current release flow. Let's create a doc first about how this would be integrated into the current workflow.

Reply:
Ok, will remove it.
Should we close this pull request?